packages/nimble-components/src/breakpoint/IxD.md

```diff
@@ -0,0 +1,72 @@
+# Nimble Component (IxD): Breakpoint
+
+## Overview
+
+A breakpoint is a UI element that visually marks a line or section of code allowing developers to pause execution and inspect the application state during debugging. In a code table or editor, breakpoints help users quickly identify and interact with debugging points.
+
+> NOTE: THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+### Background
+Requested by teams wishing to support code inspection and debugging workflows in Nimble Table.
+
+## Usage
+
+Use with the Nimble Table when displaying code, especially in environments where debugging is required.
+
+- **When should client-users not use this component?**
+  - When displaying static code with no debugging functionality.
+  - In tables not intended for code or debugging workflows.
+
+### Anatomy
+
+![Breakpoint States](./specs/spec-images/States.png)
+
+### Related Components
+
+- **Differs from:**
+  - **Toggle buttons**  
+    - Toggle buttons are used to switch between two states for a setting or feature, typically representing on/off or enabled/disabled.  
+    - Breakpoints specifically mark a location in code for debugging purposes and are contextually tied to code execution, not general UI state changes.
+
+- **Decision guidance:**
+  - Use breakpoints for debugging; use toggle buttons for general UI state changes.
+
+## Design
+
+### Configuration
+*What types of options need to be available on the component to support client-user use cases?*
+
+### Behavior
+
+- **Built-in behaviors:**
+  - Toggle breakpoint on click/tap
```

Comment:
Assuming selection is enabled, do you have guidance about whether the row should or should not become selected if you click on the cell to toggle a breakpoint?